src/client/components/TabbedContainer/index.js

```diff
@@ -90,7 +90,8 @@ function TabbedContainer() {
       }
 
       // Don't scroll down to the top of the sidebar on initial page load
-      if (initialPageLoad.current) {
+      // unless it's loading a secondary page
+      if (initialPageLoad.current && tabIndex === 0) {
```

Comment:
Hm. I'm not sure if there's a good solution to this, but it feels weird that if somebody navigates directly to http://localhost:3000/guide/after-you-submit from social media or something, it will immediately scroll down past the header. Is there an easy way to distinguish between somebody navigating to a different tab from within the application versus outside the application? If not, feel free to merge and create a ticket to look into this later.

Reply:
Good point, scrolling down on load is unusual behavior. This was only necessary if it was a normal link not using the router (not using a <Link>), and in the process of trying to figure out why that was failing tests, I figured out #129.

So thanks for asking, it's fixed and #129 is fixed too 😅

Reply:
😂 Thanks Kalvin!